### ISSUE TITLE
Make the Link Button selectable.

### ISSUE THREAD

**Issue by lijunle** [OfficeDev/office-ui-fabric-react#4429]
When render a [`Link`](https://developer.microsoft.com/en-us/fabric#/components/link) component without `href` prop, we will render it as a `<button>` tag.

However, that is a down side of `<button>` tag, the text inside the `<button>` tag cannot be selected. There is a CSS [`user-select:text`](https://css-tricks.com/almanac/properties/u/user-select/) to solve this problem.

Here is how it looks now:

> ![image](https://user-images.githubusercontent.com/1296500/38191540-c134b02a-369b-11e8-838d-a8e1adbe918e.png)
